app/services/rad_audit_search.rb

```diff
@@ -30,7 +30,8 @@ def filters_def
        { input_label: 'Action',
          column: :action,
          options: %w[create update destroy] },
-       { column: :remote_address, type: RadCommon::LikeFilter }]
+       { column: :remote_address, type: RadCommon::LikeFilter },
+       { column: :auditable_id, type: RadCommon::EqualsFilter, data_type: :string }]
```

Comment:
curious, this is an integer field but you're setting data type to string?

Reply:
you're right this should be integer. Forgot to change this back when I was testing